CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ option(GLOW_WITH_OPENCL "Build the OpenCL backend" ON)
 option(GLOW_BUILD_EXAMPLES "Build the examples" ON)
 option(GLOW_BUILD_TESTS "Build the tests" ON)
 option(GLOW_WITH_BUNDLES "Build bundles" OFF)
+OPTION(LINK_PROTOBUF_AS_DLL "Link against protobuf build as dynamic libray." OFF)
```

Comment:
All of the options before this one are spelled with lower case.

Reply:
Fixed